examples/css/index.html

```diff
@@ -82,7 +82,27 @@ <h3 class="mistica-text-title2">Tag</h3>
                     <div class="mistica-tag-warning">Warning</div>
                     <div class="mistica-tag-error">Error</div>
                 </div>
-
+                <h3 class="mistica-text-title2">Counter</h3>
+                <div class="mistica-counter">
+                    <button class="mistica-counter__button" aria-label="decrease" disabled>
+                        <svg width="20" height="20" viewBox="0 0 24 24" aria-hidden>
+                            <path
+                                fill="currentColor"
+                                d="M3.001 12.047a.88.88 0 0 1 .828-.924h16.34a.876.876 0 0 1 .003 1.752H3.829A.876.876 0 0 1 3 12.047"
+                            />
+                        </svg>
+                    </button>
+                    <input type="number" aria-label="counter" value="0" min="0" max="10" step="1" />
```

Comment:
Is possible to write in the input, is something we can block from our side?

Reply:
We can change the input to readonly (I've changed it).
But I think it would be better and more accesible if the user can edit it using the keyboard too